participants/david_leitner.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "David Leitner",
+    "company": "SQUER Solutions",
+    "when": {
+        "friday": true,
+        "saturday": false
+    },
+    "iCanTakeNotesDuringSessions": false,
+    "tags": ["React", "Svelte", "Solid", "Next", "GraphQL"],
+    "vegan": false,
+    "vegetarian": false,
+    "whatIsMyConnectionToJavascript": "Engineer",
+    "whatCanIContribute": "I talk about the state of MicroFrontends",
+    "tShirt": {
+        "size": "L",
+        "type": "regular"
+    },
+    "twitter": "@duffleit",
+    "mastodon": "@duffleit@mastodon.social",
```

Since mastodon handles and servers can be very special, we decided to use a complete URL for it. I suppose you wanted to use this link here:

```suggestion
    "mastodon": "https://mastodon.social/@duffleit",
```